next/pages/guidelines/color/palette.tsx

```diff
@@ -0,0 +1,406 @@
+    },
+    red: {
+        src: red,
+        alt: 'user interface example where red colors are applied',
+    },
+    blue: {
+        src: blue,
+        alt: 'user interface example where blue colors are applied',
+    },
+    green: {
+        src: green,
+        alt: 'user interface example where green colors are applied',
+    },
+    indigo: {
+        src: indigo,
+        alt: 'user interface example where indigo colors are applied',
+    },
+};
+
+interface ColoredPillProps {
+    fill: string;
+    title: string;
+    value: string;
+}
+
+function TokenPill({ fill, title, value }: ColoredPillProps): JSX.Element {
+    return (
+        <div>
+            <div>{title}</div>
+            <div
+                className="ph3 pv2 br-pill"
+                style={{
+                    background: `${fill}`,
+                }}
+            >
+                <span className="white">{value}</span>
+            </div>
+        </div>
+    );
+}
+
+function ColorSection({ values }: { values: Color['values'] }): JSX.Element {
+    const [active, setActive] = useState(false);
+    return (
+        <div
+            className={classNames('flex flex-column tp-body-3', {
+                white:
+                    (['400', '500', '600'].includes(values.level) &&
+                        values.color !== 'Yellow 400') ||
+                    ['Black', 'Black 300'].includes(values.color),
+            })}
+            style={{
+                background: `${values['light-hex']}`,
+            }}
+        >
+            {/* clickable region */}
+            <ClickableBox
+                className="pv3 ph3 b flex flex-row cursor-pointer"
+                onClick={(): void => setActive(!active)}
+            >
+                <div className="flex-auto tp-body-2">
+                    <span className="pr2">{values.color}</span>
+                    {values.level === '400' ? (
+                        <CoreColorPill tokenColor={values.javascript} />
+                    ) : null}
+                </div>
+
+                {values.level}
+                {!active ? <NavigationCaretDownSmall /> : <NavigationCaretUpSmall />}
+            </ClickableBox>
+            {/* end clickable region */}
+            <div
+                className={classNames('ph3', {
+                    'h-0 overflow-hidden': !active,
+                    'h-auto pb3 ': active,
+                })}
+            >
+                {/* body content */}
+                <div className="pb3">
+                    <ul style={{ listStyle: 'bullet' }} className="ml3">
+                        {values.description.split('\n').map(item => {
+                            return <li key={item.replace(' ', '-')}>{item}</li>;
+                        })}
+                    </ul>
+                </div>
+                {/* tokens */}
+                <div className="flex flex-row col-gap2 row-gap2 flex-wrap mv3">
+                    {TokenPill({
```

Comment:
This is preferred:

```jsx
<TokenPill title="Hex" value={...} />
```

Reply:
🤦‍♂️ Not sure where my head was at there. Thank you!